projects/gameboard-ui/src/app/game/player-enroll/player-enroll.component.ts

```diff
@@ -134,7 +134,10 @@ export class PlayerEnrollComponent {
 
   delete(p: Player): void {
     const sub: Subscription = this.api.delete(p.id).pipe(
-      finalize(() => sub.unsubscribe())
+      first(),
```

Comment:
I'm not understanding `first()` here?

Reply:
Oh, good to know.  Didn't realize `first()` unsubscribed.  So you don't even need the `sub` variable anymore.